docker/php-test-stepup/Dockerfile

```diff
@@ -19,14 +21,18 @@ RUN apt-get update && apt-get install -y \
 ENV NVM_DIR /usr/local/nvm
 ENV NODE_VERSION 10
 
-# Install nvm with node and npm
+# install nvm
+# https://github.com/creationix/nvm#install-script
```

Comment:
Forgot to remove a stray dev-comment?

Also, regarding line 24 and 29: Why not start a sentence with a capital letter? Just nitpicking here!